packages/cardano-services/package.json

```diff
@@ -116,6 +118,7 @@
     "json-bigint": "^1.0.0",
     "jsonschema": "^1.4.1",
     "lodash": "^4.17.21",
+    "next-json": "^0.3.2",
```

Comment:
I don't mind using this if it's better, but then I think we should be consistent and replace `toSerializableObj`/`fromSerializableObj` utils within entire codebase. Probably better to do in a separate PR, good topic for tech talks too!

```suggestion
```

Reply:
6ba4ef5a2d should solve this